src/test/java/htsjdk/samtools/SamReaderTest.java

```diff
@@ -169,4 +177,89 @@ public void samRecordFactoryNullHeaderTest() {
         Assert.assertTrue(samRec.getHeader() == null);
     }
 
+    @Test
+    public void testAssertingIteratorUsesLenientOrdering() {
+        //the coordinate comparator's strict sort sorts lower mapping qualities first
```

Comment:
`//` comment needs a space after the `//`. Also in genearl it's nice if comments are written as sentences, with capitalization and punctuation, IMO it makes them easier to read.